products/iap/api.yaml

```diff
@@ -135,3 +135,66 @@ objects:
         name: 'name'
         description: Name of the instance.
         required: true
+  - !ruby/object:Api::Resource
+    name: 'Brand'
+    base_url: 'projects/{{project}}/brands'
+    self_link: '{{name}}'
+    input: true
+    description: |
+      OAuth brand data.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Output only. Identifier of the brand. NOTE: GCP project number achieves
```

Comment:
Can we document the format of `name`? In the generated documentation we have the import format listed as just `{name}` which will be difficult to figure out if we don't list the actual format of `name` somewhere

Reply:
Done.